client/app/states/dashboard/dashboard.state.js

```diff
@@ -188,6 +188,31 @@ function StateController($state, RequestsState, ServicesState, definedServiceIds
     $state.go('services');
   };
 
+  vm.navigateToRetiredServicesList = function() {
```

Comment:
I thought about doing a function that acted in 3 different ways but opted against it.   I could possibly combine them all into one larger function but that is more of a style preference.  As for the requests dashboard links, they weren't a part of this PR.  This PR addresses a specific bug caused by clicking the services links on the dashboard.